spline-shadow/client-ui/src/app/services/lineage/lineage-graph.service.ts

```diff
@@ -52,10 +55,12 @@ export class LineageGraphService {
     let that = this
     return that.executionPlanControllerService.lineageUsingGETResponse(executionPlanId).pipe(
       map(response => {
-        return that.convertFromExecutedLogicalPlanToFromExecutedLogicalPlanViewModel(response)
+        let convertedResponse = that.convertFromExecutedLogicalPlanToFromExecutedLogicalPlanViewModel(response)
```

Comment:
convertFromExecutedLogicalPlanTo**From**ExecutedLogicalPlanViewModel
Extra "from"?
Could simplify also as "toLogicalPlanView" rest could be seen from function input n output types.

Reply:
yes. I will modify it